multinode_runfabtests.sh

```diff
@@ -24,3 +32,16 @@ if [ ${PROVIDER} == "efa" ]; then
     fi
 fi
 bash -c "$runfabtests_script ${FABTESTS_OPTS} ${PROVIDER} ${SERVER_IP} ${CLIENT_IP}"
+
+# dgram_pingpong test has been excluded during installation
+# (in install-fabtests.sh), because it does not work with "-E" option.
+# So here we run it separately using "-b" option
+echo "Run fi_dgram_pingpong with out-of-band synchronization"
+dgram_pingpong="${HOME}/libfabric/fabtests/install/bin/fi_dgram_pingpong"
+${dgram_pingpong} -k -p efa -b &
+server_pid=$!
+# sleep 1 second for server to start
+sleep 1
+ssh ${CLIENT_IP} "${dgram_pingpong} -k -p efa -b ${SERVER_IP}"
```

Comment:
Do we need to check return codes here? Or is something else validating that this passed?

Reply:
There is an `set -xe` in this script, is that enough?

Reply:
I think we should follow how it does in `function cs_test` in runfabtest.sh, where we wait for client process and get its return code; if it returns error, kill the server process.

Reply:
I was relying the `set -xe` at the beginning to check return code. But I think @zhngaj suggestion is valid, we need to kill server process if client process failed. will update.

Reply:
Done